user_guide/user_guide.md

```diff
@@ -95,20 +95,97 @@ On the Control Page the existing Routines can be stated manually by clicking the
 
 ## Devloper Suite
 
-The Developer Suite allows to create, edit and delete Functions i.e. Operations or APIs.
+The Developer Suite allows to create, edit and delete Functions or APIs. It is important to mention that Operation and Function can be used synonymous, more precisly Function is only used in the context of the Developer Suite as it behaves like a function in programming. The developer suite requires decent knowledge about several cloud technologies and can render the app to break.
 
-*Click the "Switch to Developer Suite" button.*
+It is important
 
 ![Control Page](../resources/images/control_page.png)
 
+*Click the "Switch to Developer Suite" button. Confirm the warning with the "Ok" button.*
+
+![Developer Suite](../resources/images/developer_suite.png)
+
 ### Functions
 
+Since we are already automatically in the Manage Functions mode we have to do nothing.
+
+In case you are not:
+*Click the "Manage Functions" button.*
+
 #### Create
+*Click the "Create a new Function" button.*
+
+![Create a new Function](../resources/images/create_function1.png)
+
+*Click on the switch next to "In protected mode" to make the fields editable. Click again to switch to protected mode again.*
+
+We now see a number of fields that can or have to be filled.
+
+**Category** specifies under which category the Function is listed in the Operations in the Routine creation.
+
+**Function ID** is a unique identifier for the Function that can be a custom string.
+
+**Function Label** is the label that is displayed for the Operation in the Routine creation.
+
+A **result of a Function** is optional and can be deactived in activated with the "Function returns a Result Variable" checkbox.
+
+**Result ID** is a unique identifier for the result that can be a custom string and has to be used in the API definition.
```

Comment:
"and has to be used in the API Definition" stimmt nicht so ganz. Man kann zwar das Ergebnis einer anderen Funktion benutzen, der Name der result variable spielt hierbei aber keine Rolle, da wir ja eh nur eine haben. Von daher würde ich den zweiten Teil löschen.